src/providers/contexts.ts

```diff
@@ -27,3 +27,7 @@ export const OrganizationContext = createContext<ProvidedOrganizationData>({
   },
   selectedOrganization: defaultSelectedOrg,
 });
+
+export const LocalizationContext = createContext<ProvidedTimeZones>({
```

Comment:
Ideally this should provide the time zones and the strings.
```
{
  timeZones: TimeZoneDescription[],
  strings: { [key: string]: string },
```
For now the provider can load the strings from src/strings.. when we have i18n we will get the strings from the BE.

Also, if you prefer we do this separately as part of the i18n work that's fine too.

Reply:
good call, I misunderstood that part of the requirement

Reply:
i created tickets to move user logic from App into UserProvider, same for org. I can take a crack at those after this lands.

Reply:
Btw - if you expose strings as well, we can share the load of updating the rest of the codebase as a separate effort if you prefer.